External/FEXCore/Source/Interface/Core/JIT/Arm64/BranchOps.cpp

```diff
@@ -370,6 +504,7 @@ void Arm64JITCore::RegisterBranchHandlers() {
   REGISTER_OP(JUMP,              Jump);
   REGISTER_OP(CONDJUMP,          CondJump);
   REGISTER_OP(SYSCALL,           Syscall);
+  REGISTER_OP(INLINESYSCALL,     InlineSyscall);
```

Comment:
Could you add a brief comment describing the difference between SYSCALL and INLINESYSCALL? It's probably obvious if you already know how SYSCALL is implemented, but if you look at them without prior knowledge it's difficult to see why there are two and how they differ.

Reply:
Added to IR.json